README.md

```diff
@@ -58,26 +113,36 @@ export interface NgoOptions {
 }
 ```
 
-## Webpack loader usage:
+
+## Webpack loader and plugin usage:
 
 ```
-"module": {
-  "rules": [
-    {
-      "test": /\.js$/,
-      "loader": "ngo-loader"
-      "options": {
-        sourceMap: false
+const PurifyPlugin = require('ngo').PurifyPlugin;
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        loader: 'ngo/dist/ngo-webpack-loader'
```

Comment:
you can get rid of the `dist` and achieve `ngo/webpack-loader` if you create `/webpack-loader` dir that contains just `package.json` with a single entry `{'main': '../dist/webpack-loader'}`.